pkg/kubelet/rkt/rkt.go

```diff
@@ -1069,10 +1073,16 @@ func (r *Runtime) Type() string {
 }
 
 func (r *Runtime) Version() (kubecontainer.Version, error) {
+	if err := r.getVersions(); err != nil {
+		return nil, err
+	}
 	return r.binVersion, nil
```

Comment:
@yifan-gu I didn't quite get it...It seems that the `r.binVersion` is still sharing without protection, right?